buildSrc/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ plugins {
   `java-gradle-plugin`
   groovy
   id("com.diffplug.spotless") version "5.6.1"
+  `kotlin-dsl`
```

Comment:
Is this a mistake?

Reply:
It wasn't
I added this dependency because I moved dependencies to Kotlin in `buildSrc` module